package.json

```diff
@@ -66,6 +67,7 @@
     "react-hot-loader": "^3.1.3",
     "react-page-click": "^4.0.2",
     "react-redux": "^5.0.6",
+    "react-tap-event-plugin": "2.0.1",
```

Comment:
Is this necessary? It's been deprecated and "since the problem it solves has been fixed in most browsers by now you should migrate away from this plugin." - https://www.npmjs.com/package/react-tap-event-plugin

Reply:
i was facing exception, had to add it.
```js
Warning: Unknown prop `onTouchTap` on <div> tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop
    in div (created by Overlay)
    in Overlay (created by DialogInline)
    in div (created by DialogInline)
    in DialogInline
End of output
npm ERR! Test failed.  See above for more details.```